bin/cron/create_rollup_tables

```diff
@@ -68,6 +67,7 @@ def main
           contained_level_text: sanitize_string_for_db(contained_question)
         }
 
+        next unless contained_level.type == 'Multi'
```

Comment:
`Match` and `EvaluationMulti` also have "answers". do we want to include those too?

here are the counts of the types of the sublevels of level groups:
```
[development] dashboard > LevelGroup.all.map(&:levels).flatten.compact.map(&:type).tally
=> {"Multi"=>7499, "FreeResponse"=>581, "Match"=>52, "TextMatch"=>24, "EvaluationMulti"=>74}
```

Reply:
Yes please, that would help us validate answers for assessments

Reply:
OK, I added those to the check.